client/src/components/ClubPage.tsx

```diff
@@ -141,33 +182,63 @@ export default function ClubPage() {
           isOwner={memberType === "owner"}
         />
       )}
+      {showCreateAnnModal && (
+        <AnnouncementCreationModal
+          hideModal={() => {
+            setShowCreateAnnModal(false);
+          }}
+          //requestupdate for updating announcements
+          clubId={id}
+          userId={user?.id}
+          requestUpdate={updateAnnouncementList}
+          />
+      )}
       <h2 className="club-heading">{clubName}</h2>
-      <h5>{clubDesc}</h5>
+      <h5 class="club-desc">{clubDesc}</h5>
 
       {/* Render join button if user isn't a member, otherwise render the leave button for non-owners (members, executives) */}
-      {memberType != "member" && <SettingsButton />}
-      {memberType === "none" ? (
+      {memberType != "member" && <SettingsButton/>}
+      {memberType === "none" && clubStatus === "open" ? (
         <JoinButton />
+      ) : memberType === "none" && clubStatus === "application" ? (
+        <ApplyButton/>
       ) : memberType === "owner" ? null : (
         <LeaveButton />
       )}
-      <div>
-        Members
-        {members.map((member) => {
-          return (
-            <div key={member["userId"]}>
-              {member["name"]}
-              {user?.id != member["userId"] ? ( //Don't render menu option if member is the logged in user
-                <MoreVertical
-                  onClick={() => {
-                    setSelectedMember(member);
-                    setShowMemberModal(true);
-                  }}
-                />
-              ) : null}
-            </div>
-          );
-        })}
+      <div class="first-row">
+        <div class="members">
+          <h1>Members</h1>
+          {members.map((member) => {
+            return (
+              <div key={member["userId"]}>
+                {member["name"]}
+                {user?.id != member["userId"] ? ( //Don't render menu option if member is the logged in user
+                  <MoreVertical
+                    onClick={() => {
+                      setSelectedMember(member);
+                      setShowMemberModal(true);
+                    }}
+                  />
+                ) : null}
+              </div>
+            );
+          })}
+        </div>
+        <div class="announcement-container">
+        <div class="announcement-header">
+            <h1>Announcements</h1>
+          </div>
+          <div class="scroll">
+            {announcements.map((announcement) => {
+              return (
+                <ul key={announcement["id"]} class="announcement" onClick={() => alert(announcement["announcementText"])}>
```

Comment:
Same thing - I'd avoid alerts, a Modal would be quick to do and look much better